<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1373</width>
    <height>953</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="enemyLayer">
    <property name="geometry">
     <rect>
      <x>-30</x>
      <y>0</y>
      <width>791</width>
      <height>571</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="GameCanvas" name="gameCanvas">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>700</width>
      <height>700</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="deadWorldCanvas">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>201</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="naturalKillerTowerButton">
    <property name="geometry">
     <rect>
      <x>1161</x>
      <y>280</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Natural Killer Cell</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/textures/Images/TD_NaturalKillerTower.png</normaloff>:/textures/Images/TD_NaturalKillerTower.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="memoryBTowerButton">
    <property name="geometry">
     <rect>
      <x>1251</x>
      <y>280</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Memory B Cell</string>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/textures/Images/TD_MemoryBTower.png</normaloff>:/textures/Images/TD_MemoryBTower.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="boneMarrowCountLabel">
    <property name="geometry">
     <rect>
      <x>1244</x>
      <y>201</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Bone Marrow</string>
    </property>
    <property name="text">
     <string>500</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="livesCountLabel">
    <property name="geometry">
     <rect>
      <x>1246</x>
      <y>109</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Lives</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="roundProgressLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>170</y>
      <width>161</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Round
Progress:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="levelProgressHeaderLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>60</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Round:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="levelProgressLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="totalLevelsLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>100</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="slashLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>31</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="heartImageButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>100</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Lives</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/textures/Images/heart.png</normaloff>
      <disabledoff>:/textures/Images/heart.png</disabledoff>:/textures/Images/heart.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="boneImageButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1170</x>
      <y>190</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Bone Marrow</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resources.qrc">
      <normaloff>:/textures/Images/boneMarrow.png</normaloff>
      <disabledoff>:/textures/Images/boneMarrow.png</disabledoff>:/textures/Images/boneMarrow.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="VirusProgress">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>260</y>
      <width>161</width>
      <height>441</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1160</x>
      <y>380</y>
      <width>201</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>enemyLayer</zorder>
   <zorder>gameCanvas</zorder>
   <zorder>deadWorldCanvas</zorder>
   <zorder>naturalKillerTowerButton</zorder>
   <zorder>memoryBTowerButton</zorder>
   <zorder>boneMarrowCountLabel</zorder>
   <zorder>livesCountLabel</zorder>
   <zorder>roundProgressLabel</zorder>
   <zorder>levelProgressHeaderLabel</zorder>
   <zorder>totalLevelsLabel</zorder>
   <zorder>levelProgressLabel</zorder>
   <zorder>slashLabel</zorder>
   <zorder>heartImageButton</zorder>
   <zorder>boneImageButton</zorder>
   <zorder>VirusProgress</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1373</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameCanvas</class>
   <extends>QGraphicsView</extends>
   <header>gamecanvas.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
